src/main/java/com/iuc/requests/controller/UserController.java

```diff
@@ -19,15 +18,26 @@
 
     @Autowired
     private UserService userService;
+    @Autowired
+    ModelMapper modelMapper;
 
     @GetMapping("/staff")
     public StaffDto findStaffByEmail(@RequestParam(value = "email") @Email String email) {
-        return null;
+        if(email.isEmpty()) {
+            return null;
+        } else{
+            return !userService.findStaffByEmail(email).equals(null)? userService.findStaffByEmail(email): null;
```

Comment:
What is the new? I think is the same that return ``userService.findStaffByEmail(email)``

Reply:
The comment has been taken into consideration